org-cyf-itp/layouts/_default/needs-review.html

```diff
@@ -9,6 +9,8 @@
 
   <div id="pr-list">Loading...</div>
 
+  <input type="text" id="reviewer-filter" placeholder="Filter by reviewer GitHub username" />
```

Comment:
Placeholders are not a reliable or accessible way of labelling an input.
```suggestion
  <label>Filter by reviewer <input type="text" id="reviewer-filter" placeholder="Input a GitHub name" /></label>
```